rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc

```diff
@@ -6,6 +6,8 @@ If the bot has status *SUSPENDED*, review the execution plans.
 
 If a bot has status *STOPPED*, *OVERDUE*, or *UNKNOWN*, an IT administrator needs to log on to the computer the bot is installed on and needs to troubleshoot it there.
 
+If a bot is running properly, but a process execution fails, and you need help to detect the cause, turn on debug logging. 
```

Comment:
Should bot be capitalized? You capitalize it later.

Reply:
I chose the elegant third version:)

We (the RPA docs team) decided to capitalize 'bot' only if it's used as a product name ('RPA Bot'). In all other cases we write it in lower case for better readability. Since the version belongs to the product, we use 'RPA Bot [version] 1.0.0'.